dev/pages/Badge.vue

```diff
@@ -0,0 +1,60 @@
+<script setup>
+import {computed, reactive, ref} from 'vue';
+import { wBadge } from '#components';
+import { radio } from '#dev-util';
+
+const badgeTypes = [
+  { name: 'Undefined', radio },
+  { name: 'neutral', radio },
+  { name: 'info', radio },
+  { name: 'positive', radio },
+  { name: 'warning', radio },
+  { name: 'negative', radio },
+  { name: 'disabled', radio },
+  { name: 'notification', radio },
+  { name: 'price', radio },
```

Comment:
I see that when it comes to styling/color of a component, a `variant` name is often used instead of `type` (see Button or Alert). I think it's the same way designers refer to different types of components.
Not that `variant` would be my first pick when dealing with a component type but we couldn't use `type` in e.g. Button for a similar purpose, because that was a reserved attribute there. Then we had to go with `variant`. 
I'm saying all that to hear what you think about aligning this sort of API. What we decide today might be difficult to rename later, so I thought it's better to ask now.

Reply:
Sure I can change it to `variant`. 👍